tests/ext-activitylog.test.js

```diff
@@ -81,26 +81,16 @@ test('show/hide logs associated with extension id that is checked/unchecked from
     '.toggle-btn'
   );
 
-  const extFilterCheckboxList = activityLog.view.extFilter.shadowRoot.querySelector(
-    '.checkbox-list'
-  );
-
   activityLog.handleNewLogs(logs);
 
   const tableBody = activityLog.view.logView.shadowRoot.querySelector('tbody');
   const getTableRows = () => tableBody.querySelectorAll('tr');
   let tableRows = getTableRows();
 
-  // Initially filter Checkbox List is hidden
```

Comment:
Instead of just removing it, it would be good to actually verify that the dropdown logic works as expected.

Reply:
added the test to verify the dropdown logic.